src/components/Dropdown/Dropdown.tsx

```diff
@@ -258,6 +258,7 @@ export function Dropdown(props: DropdownProps) {
     options,
     children,
     'aria-label': ariaLabel,
+    value = null,
```

Comment:
With HeadlessUI update, this complains if it's empty that it's changing from uncontrolled to controlled

Reply:
Do you think this is worth a code comment? Maybe just something about it being required by Headless UI.

Reply:
Makes sense, will add comment for both files